tutorials/Getting_Started/README.md

```diff
@@ -106,37 +104,53 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.unit.IntSize
 import androidx.compose.ui.unit.dp
 
-fun main() = Window(title = "Compose for Desktop", size = IntSize(300, 300)) {
-    val count = remember { mutableStateOf(0) }
+fun main() = Window(
+        title = "Compose for Desktop", 
+        size = IntSize(300, 300)
+        ) {
+    val count by remember { mutableStateOf(0) }
     MaterialTheme {
-        Column(Modifier.fillMaxSize(), Arrangement.spacedBy(5.dp)) {
-            Button(modifier = Modifier.align(Alignment.CenterHorizontally),
+        Column(
+            modifier = Modifier.fillMaxSize(), 
+            // spacedBy() sets the distance between each Composable
+            Arrangement.spacedBy(5.dp)
+        ) {
+            Button(
+                modifier = Modifier.align(Alignment.CenterHorizontally),
                    onClick = {
+                       // Becareful using .value++, it can cause un needed Recomposes
```

Comment:
// Be careful when using .value++, it can cause unneeded Recomposes
  
  
  is 'Recomposes' right?

Reply:
@B1ggDave  Do you think Recomposing would be more grammatically correct?

Reply:
Used Recomposing